src/components/loading/loading-controller.ts

```diff
@@ -35,6 +35,11 @@ import { LoadingOptions } from './loading-options';
  * `onDidDismiss` function can be called to perform an action after the loading
  * indicator is dismissed.
  *
+ * Attempting to dismiss a loading indicator before it has been completely
+ * presented will result in errors, so make sure to enclose all code paths
```

Comment:
The exception is as follows: 
> Uncaught, in promise

Which really means, that `loader.dismiss()` rejected.
If you have a catch, for example:
`loader.dismiss().then(() => {}, (err) => console.log(err))`
It will just `console.log` with `false`, but if you don't have a `catch`, it will throw this as an error.